srcpkgs/lv2/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'lv2'
 pkgname=lv2
 version=1.14.0
-revision=1
-lib32disabled=yes
+revision=2
 build_style=waf
-homepage="http://lv2plug.in"
 hostmakedepends="python"
 makedepends="libsndfile-devel gtk+-devel"
 short_desc="Plugin standard for audio systems"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="LGPL-2.1, BSD"
```

Comment:
please fix license

Reply:
Didn’t change as I don’t recognize license in file https://github.com/drobilla/lv2/blob/master/COPYING  Didn’t want to change it without knowning the correct one to change it to